src/App.css

```diff
@@ -1,38 +1,80 @@
-.App {
-  text-align: center;
+a {
+  text-decoration: none;
+  color: black;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.app-header,
+.footer {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: #2b4570;
+  color: white;
+}
+
+/* .footer {
+  margin-top: auto;
+} */
+
+.footer-h-cont {
+  margin-inline: auto;
+}
+
+.home-r {
+  color: white;
+  padding-left: 15px;
+}
+
+.nav-branding {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+}
+
+.nav-brand-title h2,
+.nav-branding {
+  padding-right: 10px;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.city-img {
+  width: 279px;
+  height: 173px;
+  border-radius: 8px;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+.main-div {
   display: flex;
   flex-direction: column;
   align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+  background-color: #a8d0db;
+  padding-bottom: 20px;
+  min-height: 90vh;
+  overflow: scroll;
+}
+
+/* body {
+  background-color: #a8d0db;
+  display: flex;
+  flex-direction: column;
+} */
```

Comment:
- [ ] Please, kindly remove this old piece of commented out code, in order to improve the cleanliness of the repo 💪